README.md

```diff
@@ -122,6 +123,23 @@ To upload stuff to S3, you'll need credentials. Your **AWS credentials** can be
 - If we can't find these, we'll look for an `.aws/config` file in the local environment.
 - NOTE: If the bucket is public, no credentials are necessary.
 
+### Dropbox
+Dropbox storage handler needs a user access token. For this:
+- Create Dropbox account (if you don't have).
+- Go to https://dropbox.com/developers
+- Create a new application with App Folder (full access is not recommended, because without directory defined in the configuration it will have destructive results).
+
+You can also define a custom location (root is App Folder) using 
+`upload_directory` optional parameter. This **should** begin with slash
+and **must** end with slash. Default is root. Note that all files in this directory will
+be deleted after `retention_days`.
+
+The configuration connections strings may look like the following:
```

Comment:
```suggestion
The Dropbox storage handler needs a user access token in order to work. To get one, do the following:
- Create a Dropbox account (if you don't already have one).
- Go to https://dropbox.com/developers
- Create a new application with App Folder access. **Do not give it full access**, as this may have dangerous, destructive consequences if configured incorrectly.

You can also define a custom location (root is App Folder) using the
`upload_directory` optional parameter. This **should** begin with slash
and **must** end with slash. Default is root.

The configuration connections strings may look like the following:
```

> Note that all files in this directory will be deleted after `retention_days`

This is not the intention - we should only be deleting the files that match a certain regex. That regex should match our naming scheme, e.g. if the backup file is named `postgres-2020-01-01.backup` then we should only be matching with something like `r"postgres-\d{4}-\d{2}-\d{2}.backup"`.

I haven't implemented this quite yet, though. I'll open an issue for it.